segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperLog.java

```diff
@@ -477,7 +478,11 @@ private boolean executeWrites(List<Write> toExecute) {
                 }
 
                 // Invoke the BookKeeper write.
-                w.getWriteLedger().ledger.asyncAddEntry(w.data.array(), w.data.arrayOffset(), w.data.getLength(), this::addCallback, w);
+                final WriteHandle handle = w.getWriteLedger().ledger;
```

Comment:
You don't actually need to store this in a variable as you're only using it once. You can just invoke `appendAsync` from `w.getWriteLedger().ledger` directly.

Reply:
okay. There is no difference at runtime.
It is a matter of taste.
changed